sig-contributor-experience/annual-report-2020.md

```diff
@@ -0,0 +1,141 @@
+- Does the group have contributors from multiple companies/affiliations? Can end
+  users/companies contribute in some way that they currently are not?
+  - Yes, we have multiple avenues for contributing for both code and non-code
+    projects alike.
+ 
+
+#### Current initiatives and project health
+
+- What are initiatives that should be highlighted, lauded, shout outs, that
+  your group is proud of? Currently underway? What are some of the longer tail
+  projects that your group is working on?
+  - The contributor website (https://k8s.dev)
+  - Triage process improvements
+  - The 2020 Contributor Celebration
+  - Retired the Kubernetes-Incubator Org
+  - In Progress: Migrating the default branch on GitHub from `master` to `main`
+  - In Progress: Revamping the prow approval plugin to support granual approvals
+- Year to date KEP work:
+  - [KEP-1553: Issue Triage Workflow and Automation][kep]
+  - In Progress: KEP for revamping the prow approval plugin
+- What initiatives are you working on that aren't being tracked in KEPs?
+
+  |         Subproject        |          Initiative / Program          |
+  |:-------------------------:|:--------------------------------------:|
+  | Community                 | Community Repo Stewardship             |
+  | Community Management      | Annual Contributor Survey              |
+  | Community Management      | Calendar Admin                         |
+  | Community Management      | Chair and TL Meetings + Docs           |
+  | Community Management      | Discuss Admin                          |
+  | Community Management      | Mailing List Admin                     |
+  | Community Management      | Slack Admin                            |
+  | Community Management      | Zoom / YouTube Admin                   |
+  | Community Management      | Zoom / YouTube Automation (zapier)     |
+  | Contributor Documentation | Contributor Guide Stewardship          |
+  | Contributor Documentation | Contributor Site                       |
+  | Contributor Documentation | Developer Guide Audit                  |
+  | Contributor Documentation | Developer Guide Stewardship            |
+  | Contributor Documentation | Season of Docs                         |
+  | Contributors Comms        | Contributor / SIG Profiling            |
+  | Contributors Comms        | Stewardship of k8scontributors twitter |
+  | Devstats                  | Devstats Dashboard Update              |
+  | Events                    | Monthly Community Meeting              |
+  | Events                    | Office Hours                           |
+  | GitHub Management         | GitHub Admin / Moderation              |
+  | GitHub Management         | GitHub Master -> Main rename           |
+  | GitHub Management         | GitHub New Membership Coordinator      |
+  | Mentoring                 | 1:1 Hour                               |
+  | Mentoring                 | Google Summer of Code                  |
```

Comment:
+ Google Season of Docs (it's mostly a SIG-Docs thing, but still lands in the ContribX mentoring area).

Reply:
I moved it from mentoring to contributor docs because its no longer a direct mentoring program. Instead they will let you contract your own tech writers.